consensus/types/src/beacon_block.rs

```diff
@@ -67,97 +67,11 @@ impl<T: EthSpec> BeaconBlock<T> {
 
     /// Return a block where the block has maximum size.
     pub fn full(spec: &ChainSpec) -> BeaconBlock<T> {
```

Comment:
I wonder if `BeaconBlock::full` should be named differently seeing as it's only used for tests and _shouldn't_ be used for determining the size limit on RPC calls (because it will only return an Altair block for Altair-genesis networks). How would you feel about moving this function to `beacon_node/eth2_libp2p/tests/rpc_tests.rs`, or using `BeaconBlock::empty` in that test instead?

Reply:
Agree. Will remove `BeaconBlock::full`. Will use the `full` method on the Base and Altair variants to figure out the rpc size limits.